<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="6" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-24T21:21:41 IST" name="devdomain" finished-at="2024-07-24T21:21:41 IST" duration-ms="87">
    <groups>
      <group name="Regression">
        <method signature="DevDomainTest.testMethod3()[pri:0, instance:devdomain.DevDomainTest@1622f1b]" name="testMethod3" class="devdomain.DevDomainTest"/>
        <method signature="DevDomainTest2.testMethod3()[pri:0, instance:devdomain.DevDomainTest2@4b0b0854]" name="testMethod3" class="devdomain.DevDomainTest2"/>
      </group> <!-- Regression -->
      <group name="Smoke">
        <method signature="DevDomainTest.testMethod1()[pri:0, instance:devdomain.DevDomainTest@1622f1b]" name="testMethod1" class="devdomain.DevDomainTest"/>
        <method signature="DevDomainTest.testMethod3()[pri:0, instance:devdomain.DevDomainTest@1622f1b]" name="testMethod3" class="devdomain.DevDomainTest"/>
        <method signature="DevDomainTest2.testMethod1()[pri:0, instance:devdomain.DevDomainTest2@4b0b0854]" name="testMethod1" class="devdomain.DevDomainTest2"/>
        <method signature="DevDomainTest2.testMethod3()[pri:0, instance:devdomain.DevDomainTest2@4b0b0854]" name="testMethod3" class="devdomain.DevDomainTest2"/>
      </group> <!-- Smoke -->
    </groups>
    <test started-at="2024-07-24T21:21:41 IST" name="Smoke Tests" finished-at="2024-07-24T21:21:41 IST" duration-ms="87">
      <class name="devdomain.DevDomainTest">
        <test-method signature="testMethod1()[pri:0, instance:devdomain.DevDomainTest@1622f1b]" started-at="2024-07-24T21:21:41 IST" name="testMethod1" finished-at="2024-07-24T21:21:41 IST" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod1 -->
        <test-method signature="testMethod3()[pri:0, instance:devdomain.DevDomainTest@1622f1b]" started-at="2024-07-24T21:21:41 IST" name="testMethod3" finished-at="2024-07-24T21:21:41 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod3 -->
      </class> <!-- devdomain.DevDomainTest -->
      <class name="devdomain.DevDomainTest2">
        <test-method signature="testMethod1()[pri:0, instance:devdomain.DevDomainTest2@4b0b0854]" started-at="2024-07-24T21:21:41 IST" name="testMethod1" finished-at="2024-07-24T21:21:41 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod1 -->
        <test-method signature="testMethod3()[pri:0, instance:devdomain.DevDomainTest2@4b0b0854]" started-at="2024-07-24T21:21:41 IST" name="testMethod3" finished-at="2024-07-24T21:21:41 IST" duration-ms="5" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at devdomain.DevDomainTest2.testMethod3(DevDomainTest2.java:32)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod3 -->
      </class> <!-- devdomain.DevDomainTest2 -->
    </test> <!-- Smoke Tests -->
  </suite> <!-- devdomain -->
</testng-results>
